Formula/alluxio.rb

```diff
@@ -20,6 +24,7 @@ def default_alluxio_conf
   def install
     doc.install Dir["docs/*"]
     libexec.install Dir["*"]
+    bin.env_script_all_files libexec/"bin", JAVA_HOME: Formula["openjdk@11"].opt_prefix
```

Comment:
Why not allow this to be overridable?

Reply:
yeah, I was not very sure both the syntax.

But yeah, I should do that.